hosts

```diff
@@ -360,3 +360,4 @@ lib-sc-staging1.princeton.edu
 lib-sc-prod1.princeton.edu
 [vireo_production]
 vireo-prod1.princeton.edu
+lib-proc4.princeton.edu
```

Comment:
Right now this is adding `lib-proc4` to the `vireo_production` group. I don't think that's what you intended. It won't matter for the studio proc role, as currently set up, since you're calling the individual host by name. But when someone runs the vireo role we may see weird behavior on the studio processing server.

I'm not normally a fan of single-server groups, but in this case, the name of the server doesn't immediately tell me what it's for (library processes could mean a lot of things), so I'd create a group for it.